openslides_backend/action/user/reset_password_to_default.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, Dict
+
+from ...models.models import User
+from ...shared.patterns import FullQualifiedId
+from ..default_schema import DefaultSchema
+from ..generics import UpdateAction
+from ..register import register_action
+
+
+@register_action("user.reset_password_to_default")
+class UserResetPasswordToDefaultAction(UpdateAction):
+    """
+    Action to reset a password to default of a user.
+    """
+
+    model = User()
+    schema = DefaultSchema(User()).get_update_schema()
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Gets the default_password and reset password.
+        """
+        user = self.database.get(
+            FullQualifiedId(self.model.collection, instance["id"]), ["default_password"]
+        )
+        default_password = user.get("default_password")
+        instance["password"] = default_password
```

Comment:
`default_password` password contains the clear text pw. Please hash it first

Reply:
Done.